src/popup/router/routes.js

```diff
@@ -327,6 +327,7 @@ export default [
       closeButton: true,
       hideNotificationsIcon: true,
       ifNotAuth: true,
+      scrollable: true,
```

Comment:
All pages are scrollable. This is about making the scrollbar visible. So maybe `displayScrollbar` or `scrollbarVisible` would be better here?